pdns/recursordist/docs/settings.rst

```diff
@@ -1838,14 +1838,16 @@ IP address for the webserver to listen on.
 
 ``webserver-allow-from``
 ------------------------
--  IP addresses, comma separated
+-  IP ranges, comma separated
```

Comment:
range would be something like 127.0.0.1-127.0.0.5, this should probably use word like prefix or something else?

Reply:
"IP ranges as netmasks"?

Reply:
I am copied some words from other "allow-from" settings. But I think the term IP range is confusing. I'll change the others as well.